liana-gui/src/app/state/active.rs

```diff
@@ -7,24 +7,88 @@ use super::{Cache, Menu, State};
 use crate::app::{message::Message, view, wallet::Wallet};
```

Comment:
It's not ideal having all these Panels in the same source file. Maybe have a `../state/active/mod.rs` and `../state/active/overview.rs` `../state/active/send.rs` `../state/active/revieve.rs` and `../state/active/transactions.rs`,

Reply:
Yes. I believe Andika has already gone this way in his branch, so I just left it alone here so as not to create a merge conflict.